### ISSUE TITLE
buildkite build diff-checks 22408

### ISSUE THREAD

**Issue by tmroeder** [google/llvm-premerge-checks#268]
**Describe the bug**
The build infrastructure failed to create a patch to start the CI process. It suggested creating bug here.

Relevant links: https://buildkite.com/llvm-project/diff-checks/builds/22408#a833cc29-0edd-4bf2-a7dd-240c62a82d3e

**To Reproduce**
1. Try to rerun the patch steps from the link (I can't actually do this, since I don't have access to the infrastructure)

**Expected behavior**
Uploading my patch in Phabricator should have started the build process, but the initial patching failed.

**Screenshots**
The errors seem suspicious, because this is the nth time I've uploaded a patch to for this particular review, and all the previous ones have worked.

Here is the relevant part of the log, taken directly from the link above.

$ trap 'kill -- $$' INT TERM QUIT; pip install -q -r scripts/requirements.txt
  | scripts/apply_patch.sh
  | INFO    repository exist, will reuse
  | INFO    working dir /mnt/disks/ssd0/agent/llvm-project-fork
  | INFO    Syncing local, origin and upstream...
  | INFO    Backing off reset_repository(...) for 1.1s (git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  | cmdline: git push origin main
  | stderr: 'ERROR: You must verify your email address.
  | See https://github.com/settings/emails.
  |  
  | fatal: Could not read from remote repository.
  |  
  | Please make sure you have the correct access rights
  | and the repository exists.')

**Comment by joker-eph** [comment]
Did you use `arc` to create your patch?

**Comment by ayermolo** [comment]
I have the same issue. I pasted diff directly in to llvm reviews web interface.

**Comment by KareemErgawy-TomTom** [comment]
As stated in the duplicated issue #269, I face the same error.
Patch: https://reviews.llvm.org/D92919
CI log: https://buildkite.com/llvm-project/diff-checks/builds/22472.

I created the patch with `arc`: `arc diff origin/main`

**Comment by tmroeder** [comment]
> Did you use `arc` to create your patch?

Yes, "arc diff HEAD~", which is what I've used for the other patches in the review (https://reviews.llvm.org/D92600).

**Comment by tmroeder** [comment]
Following up: just uploaded the next version of this patch (still using arc), and I'm still getting the same problem: https://buildkite.com/llvm-project/diff-checks/builds/22547#ac8ec450-febe-4f93-a200-5d9bd709bb2d

**Comment by metaflow** [comment]
See #270 The issue was caused by github asking buildbot to confirm email :robot: Now issue seems to be solved and I will keep an eye on it. Thank you for the report!
